libplatsupport/plat_include/polarfire/platsupport/plat/mstimer.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020, DornerWorks
+ *
+ * This software may be distributed and modified according to the terms of
+ * the BSD 2-Clause license. Note that NO WARRANTY is provided.
+ * See "LICENSE_BSD2.txt" for details.
+ *
+ * @TAG(DORNERWORKS_BSD)
```

Comment:
Could you make these an `SPDX-License-Identifier`, please?

Reply:
Done